tests/upgrade/0.8.2-0.9.0/jiva/suite_test.go

```diff
@@ -0,0 +1,211 @@
+
+	By("creating above storageclass")
+	_, err = ops.SCClient.Create(scObj)
+	Expect(err).To(BeNil(), "while creating storageclass {%s}", scObj.Name)
+
+	By("building a pvc")
+	pvcObj, err = pvc.NewBuilder().
+		WithName(pvcName).
+		WithNamespace(nsName).
+		WithStorageClass(scName).
+		WithAccessModes(accessModes).
+		WithCapacity(capacity).Build()
+	Expect(err).ShouldNot(
+		HaveOccurred(),
+		"while building pvc {%s} in namespace {%s}",
+		pvcName,
+		nsName,
+	)
+
+	By("creating above pvc")
+	_, err = ops.PVCClient.WithNamespace(nsName).Create(pvcObj)
+	Expect(err).To(
+		BeNil(),
+		"while creating pvc {%s} in namespace {%s}",
+		pvcName,
+		nsName,
+	)
+
+	By("verifying controller pod count ")
+	controllerPodCount := ops.GetPodRunningCountEventually(nsName, ctrlLabel, replicaCount)
+	Expect(controllerPodCount).To(Equal(replicaCount), "while checking controller pod count")
+
+	By("verifying replica pod count ")
+	replicaPodCount := ops.GetPodRunningCountEventually(nsName, replicaLabel, replicaCount)
+	Expect(replicaPodCount).To(Equal(replicaCount), "while checking replica pod count")
+
+	By("verifying status as bound")
+	status := ops.IsPVCBound(pvcName)
+	Expect(status).To(Equal(true), "while checking status equal to bound")
+
+	// TODO
+	// By("applying openebs 0.9.0")
+	// applyYAMLFromURL("https://openebs.github.io/charts/openebs-operator-0.9.0-RC3.yaml", "")
+
+})
+
+var _ = AfterSuite(func() {
+
+	By("deleting above pvc")
+	err := ops.PVCClient.Delete(pvcName, &metav1.DeleteOptions{})
+	Expect(err).To(
+		BeNil(),
+		"while deleting pvc {%s} in namespace {%s}",
+		pvcName,
+		nsName,
+	)
+
+	By("verifying controller pod count")
+	controllerPodCount := ops.GetPodRunningCountEventually(nsName, ctrlLabel, 0)
+	Expect(controllerPodCount).To(Equal(0), "while checking controller pod count")
+
+	By("verifying replica pod count")
+	replicaPodCount := ops.GetPodRunningCountEventually(nsName, replicaLabel, 0)
+	Expect(replicaPodCount).To(Equal(0), "while checking replica pod count")
+
+	By("deleting storageclass")
+	err = ops.SCClient.Delete(scName, &metav1.DeleteOptions{})
+	Expect(err).To(BeNil(), "while deleting storageclass {%s}", scObj.Name)
+
+})
+
+func applyYAMLFromURL(url string, flag string) {
+	// Read yaml file from the url
+	o, err := fetch(url)
+	Expect(err).ShouldNot(HaveOccurred())
+	defer o.Close()
+	b, err := ioutil.ReadAll(o)
+	Expect(err).ShouldNot(HaveOccurred())
+	// Connvert the yaml to unstructured objects
+	yamlString := string(b)
+	artifactString := artifacts.Artifact(yamlString)
+	artifactList, errs := artifacts.GetArtifactsListUnstructured(artifactString)
+	Expect(errs).Should(HaveLen(0))
+	// Applying unstructured objects
+	for i, artifact := range artifactList {
+		if flag == "job" && i == 0 {
+			unstructured.SetNestedStringMap(artifact.Object, data, "data")
+		}
+		if flag == "job" && i == 1 {
+			artifact.SetNamespace("default")
+		}
+		cu := k8s.CreateOrUpdate(
```

Comment:
Let us avoid use of k8s. It will be deprecated

Reply:
What alternative can be used in place of k8s ?

Reply:
Does this related to create or update.
let us avoid CreateOrUpdate since it was meant for installer package only.
Can `https://github.com/openebs/maya/tree/master/pkg/unstruct/v1alpha2` handle this requirement?